quicktest_test.go

```diff
@@ -511,6 +568,39 @@ func TestCRunDefer(t *testing.T) {
 	c.Assert(outerDefer, qt.Equals, 0)
 }
 
+type customT struct {
+	*testing.T
+	data int
+}
+
+func (t *customT) Run(name string, f func(*customT)) bool {
+	return t.T.Run(name, func(t1 *testing.T) {
+		f(&customT{t1, t.data})
+	})
+}
+
+func TestCRunCustomType(t *testing.T) {
+	ct := &customT{t, 99}
+	c := qt.New(ct)
+	called := 0
```

Comment:
`var called bool` and then just `if !called` below?

Reply:
I prefer the int because that checks that it's called exactly once.